sys/include/event.h

```diff
@@ -0,0 +1,218 @@
+/*
+ * Copyright (C) 2017 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    sys_event Event Queue
+ * @ingroup     sys
+ * @brief       Provides an Event loop
+ *
+ * This module offers an event queue framework not unlike libevent or libuev.
```

Comment:
Maybe make it a little easier on non-native speakers and remove the double negative.

Reply:
ok